gcp-skeleton-archetype/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,4 @@
+gcp-skeleton
+=============================
+
+This is a generated application from the gcp-skeleton archetype.
```

Comment:
This is a generated App Engine Standard Java
application from the gcp-skeleton archetype.